examples/typescript/package.json

```diff
@@ -6,11 +6,14 @@
   "license": "MIT",
   "dependencies": {
     "@ethersproject/wallet": "^5.7.0",
-    "@vocdoni/sdk": "0.0.12",
+    "@vocdoni/sdk": "^0.0.12",
```

Comment:
We want to have here a fixed version because it can be breaking for the moment as an alpha version. In this case I would discard this directly and update it once we have a new version published with the proposed changes in the branch